core/store/src/db/testdb.rs

```diff
@@ -80,6 +80,9 @@ impl Database for TestDB {
                     db[col].remove(&key);
                 }
                 DBOp::DeleteAll { col } => db[col].clear(),
+                DBOp::DeleteRange { col, from, to } => {
+                    db[col].retain(|key, _| *key < from || *key >= to);
```

Comment:
```suggestion
                    db[col].retain(|key, _| !(from..to).contains(*key));
```

may be easier to understand at a glance, at the very least by more directly encoding the way this range is described to be interpreted in the documentation of `DeleteRange` (i.e. that it does not include `to`)

Reply:
that is pretty neat!